manifests/components/04f_dex-server-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dex-server
+spec:
+  selector:
+    matchLabels:
+      app: dex-server
+  template:
+    metadata:
+      labels:
+        app: dex-server
+    spec:
+      serviceAccountName: argocd-server
```

Comment:
dex don't talk to kubernetes, so we can remove serviceAccountName.

Reply:
please disregard this comment. we need to launch `/shared/argocd-util` which reads configmaps, so we need separate service account, role and binding.